packages/lage/RELEASE.md

```diff
@@ -0,0 +1,17 @@
+# Release v2.0.0
+
+Version 2 of lage is finally arriving at the dist-tag of @latest! There had been many, many changes under the hood to make this release truly unique. Here are some of the highlights:
+
+New Features:
+
+* `worker_threads` based worker pool to manage the parallel targets
+* new `worker` target runner that can keep state from task to task to speed up TypeScript, ESLint, etc.
+* worker pool that allows allocation of certain number of workers per task type
+* customizable CLI argument to be passed along to the `npm script` runner
+* weighted targets to allow better utilization of resources for tasks that spawn multiple processes like Jest
+* brand new UI for an local interactive view of work done by workers
```

Comment:
The new UI is a clear easy-to-understand ergonomics win. Maybe that should go up top with a screenshot or gif.